src/cpp/session/SessionModuleContext.cpp

```diff
@@ -1070,6 +1070,25 @@ FilePath findProgram(const std::string& name)
    }
 }
 
+bool addTinytexToPathIfNecessary()
+{
+   if (isPdfLatexInstalled())
+      return false;
+
+   FilePath binPath;
+
+#if defined(_WIN32)
+   FilePath appData(core::system::getenv("APPDATA"));
```

Comment:
Is this guaranteed to be set correctly? (wondering if we should use `::SHGetKnownFolderPath`)

Reply:
Good idea -- and now that I think of it, you've already implemented this as part of the XDG support for config files! I'll make a change here.